File: Data Structures & Algorithms/duplicate-integer/submission-0.java
class Solution {
    public boolean hasDuplicate(int[] nums) {
        for (int i = 0; i < nums.length; i ++){
            for (int j = 0; j < nums.length; j ++){
                if (nums[i] == nums[j] && i != j){
                    return true;
                }
            }
        }
        return false;
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.py
class Solution:
    def productExceptSelf(self, nums: List[int]) -> List[int]:
        ret = []
        for i in range(len(nums)):
            val = 1
            for j in range(len(nums)):
                if j != i:
                    val *= nums[j]
            ret.append(val)
        return ret


        